<?xml version="1.0" encoding="UTF-8"?>

<!DOCTYPE msg
  PUBLIC "-//IBM//DTD DITA Message Reference//EN" "msgRef.dtd">
<msg xml:lang="en" id="CWLNC2169">
   <msgId outputclass="msgId">
      
      <msgNumber>CWLNC2169</msgNumber>
   
   </msgId>
   <titlealts>
      
      <searchtitle>2169</searchtitle>
   
   </titlealts>
   <msgText>A call to commit() for a transaction on a managed connection failed due to an exception when committing the physical session. Managed connection: {0}</msgText>
   <msgBody>
      <msgExplanation>

      
         <p>An unexpected error occurred when committing the physical session.</p>

   
      </msgExplanation>
      <msgUserResponse>

      
         <p>Check the caused by information in the stack trace for this exception.</p>

   
      </msgUserResponse>
      <msgOther>
      
         <p>
         
            <b>Category: </b>JmsClient</p>
   
      </msgOther>
   </msgBody>
</msg>